taxi/forms.py

```diff
@@ -10,11 +10,12 @@ class CarForm(forms.ModelForm):
     drivers = forms.ModelMultipleChoiceField(
         queryset=get_user_model().objects.all(),
```

Comment:
The queryset for the `drivers` field in `CarForm` uses `get_user_model().objects.all()`. If your user model includes non-driver users, consider filtering this queryset to include only users who are drivers. For example, you might want to filter by a specific field that indicates a user is a driver.